### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by harshjk**

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @harshjk, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;operating-system#930).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
 
-Home Assistant Operating System uses Docker as Container engine. It by default deploys the Home Assistant Supervisor as a container. Home Assistant Supervisor in turn uses the Docker container engine to control Home Assistant Core and Add-Ons in separate containers. Home Assistant Operating System is **not** based on a regular Linux distribution like Ubuntu. It is built using [buildroot](https://buildroot.org/) and it is optimized for running Home Assistant, especially on single board compute (SBC) devices like the Pi, ODROID, NUC and Tinker Board (see supported hardware below).
+Home Assistant Operating System uses Docker as Container engine. It by default deploys the Home Assistant Supervisor as a container. Home Assistant Supervisor in turn uses the Docker container engine to control Home Assistant Core and Add-Ons in separate containers. Home Assistant Operating System is **not** based on a regular Linux distribution like Ubuntu. It is built using [buildroot](https://buildroot.org/) and it is optimized for running Home Assistant, especially on single board compute (SBC) devices like the Pi, ODROID, NUC, and Tinker Board (see supported hardware below).
```
Comment (frenck) [inline README.md:5]: Adding this comma is incorrect. We don't use the oxford comma in the documentation for our project.

See: <https://developers.home-assistant.io/docs/documenting/standards#general-documentation>

File: README.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ If you're interested in finding out more about Home Assistant Operating System a
 - **Security:**
   - [AppArmor](https://apparmor.net/) Linux kernel security module
 
-If you don't have experience with these, embedded systems, buildroot or the build process for Linux distributions, then please read up on these topics. The rest of the documentation in this project is for developers and assumes you have experience of embedded systems or a strong understanding of the internal workings of operating systems.
+If you don't have experience with these, embedded systems, buildroot, or the build process for Linux distributions, then please read up on these topics. The rest of the documentation in this project is for developers and assumes you have experience with embedded systems or a strong understanding of the internal workings of operating systems.
```
Comment (frenck) [inline README.md:48]: Adding this comma is incorrect. We don't use the oxford comma in the documentation for our project.

See: <https://developers.home-assistant.io/docs/documenting/standards#general-documentation>

**Review by frenck — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (harshjk) [reply README.md]: @frenck Thanks for the review. Make sense, updated in [2c3d53c8c430e9ced0cba84dde98197090039111](https://github.com/home-assistant/operating-system/pull/930/commits/2c3d53c8c430e9ced0cba84dde98197090039111)

File: README.md (hunk 2 above)
Comment (harshjk) [reply README.md]: @frenck Thanks for the review. Make sense, updated in [2c3d53c8c430e9ced0cba84dde98197090039111](https://github.com/home-assistant/operating-system/pull/930/commits/2c3d53c8c430e9ced0cba84dde98197090039111)

**Review by frenck — APPROVED** [review]
Thanks, @harshjk 👍

**Pull request merged**